Quaver.Shared/Screens/Edit/Plugins/Timing/EditorScrollVelocityPanel.cs

```diff
@@ -186,7 +196,31 @@ private void DrawRemoveButton()
                         SelectedScrollVelocities.Add(point);
                 }
 
-                NeedsToScroll = true;
+                NeedsToScrollToFirstSelectedSv = true;
+            }
+        }
+
+        /// <summary>
+        /// </summary>
+        private void DrawSelectCurrentSVButton()
+        {
+            if (ImGui.Button("Select current SV"))
+            {
+                var currentPoint = Screen.WorkingMap.GetScrollVelocityAt(Screen.Track.Time);
+                if (currentPoint != null)
+                {
+                    SelectedScrollVelocities.Add(currentPoint);
```

Comment:
Add a check if it's already in the list before adding it. This will add multiple of the same object if you keep clicking it. Same applies for both panels.